File: 05_multipule_inheritance.py
class Class1:
    def show1(self):
        print('Hello, I am class 1')
    
class Class2:
    def show2(self):
        print('Hello,I amd class 2')

class Class3(Class1,Class2):
    def show3(self):
        print('Hello, i am class 3')


child_obj = Class3()
child_obj.show1()
child_obj.show2()
child_obj.show3()

File: 05_multipule inheritance.cpp
#include <iostream>
using namespace std;

class Class1 {
public:
    void show1() {
        cout << "I am class 1" << endl;
    }
};

class Class2 {
public:
    void show2() {
        cout << "I am class 2" << endl;
    }
};

class Class3 : public Class1, public Class2 {
public:
    void show3() {
        cout << "I am class 3" << endl;
    }
    
    void showAll() {
        Class1::show1();
        Class2::show2();
        show3();
    }
};

int main() {
    Class3 child_obj;
    child_obj.showAll();

    return 0;
}
